modules/micro/src/lib.rs

```diff
@@ -4,13 +4,18 @@
 //
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
+//! Minimal module, changes a static number and doesn't expose any functionality
+//! to the host.
```

Comment:
This module *does* expose functionality to the host. That's what `change` does. Maybe consider:
```suggestion
//! Minimal module, allows for changing a static number and doesn't expose any
//! other functionality to the host.
```